internal/server/handlers.go

```diff
@@ -63,7 +63,7 @@ func CreateToken(c *gin.Context, r *api.EmptyRequest) (*api.CreateTokenResponse,
 	if rCtx.Authenticated.User == nil {
 		return nil, fmt.Errorf("no authenticated user")
 	}
-	token, err := data.CreateIdentityToken(rCtx.DBTxn, rCtx.Authenticated.User.ID)
+	token, err := data.CreateIdentityToken(rCtx.DBTxn, rCtx.Authenticated.Organization, rCtx.Authenticated.User)
```

Comment:
This is great! Being able to use the authenticated data in this handlers and the one below instead of querying for data we already have in memory.